jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -198,6 +187,19 @@ def user_api(self):
 
         return PluginUserApi(self, expose=expose)
 
+    @observe('dataset_items')
+    def _update_disabled_msg(self, msg={}):
+        for data in self.app.data_collection:
+            if len(data.data.shape) == 3:
+                self.disabled_msg = ''
+                break
+        else:
+            # no cube-like data loaded.  Once loaded, the parser will unset this
+            self.disabled_msg = (
+                f"{self.__class__.__name__} requires a single dataset to be loaded, "
+                "please load data to enable this plugin."
```

Comment:
This would be a good time to update this to be more informative/correspond better with what is actually being checked. I'd change it to mention that the plugin is expecting a _three dimensional_ dataset to be loaded.

Reply:
Good point, I'll make that change.